guides/source/getting_started.md

```diff
@@ -7,840 +7,839 @@ This guide covers getting up and running with Ruby on Rails.
-needed.
-
-By following along with this guide, you'll create a Rails project called
-`blog`, a (very) simple weblog. Before you can start building the application,
-you need to make sure that you have Rails itself installed.
-
-NOTE: The examples below use `$` to represent your terminal prompt in a UNIX-like OS,
-though it may have been customized to appear differently. If you are using Windows,
-your prompt will look something like `C:\source_code>`.
-
-### Installing Rails
-
-Before you install Rails, you should check to make sure that your system has the
-proper prerequisites installed. These include:
-
-* Ruby
-* SQLite3
-
-#### Installing Ruby
-
-Open up a command line prompt. On macOS open Terminal.app; on Windows choose
-"Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a
-dollar sign `$` should be run in the command line. Verify that you have a
-current version of Ruby installed:
-
-```bash
-$ ruby --version
-ruby 3.2.0
-```
-
-Rails requires Ruby version 3.2.0 or later. It is preferred to use the latest Ruby version.
-If the version number returned is less than that number (such as 2.3.7, or 1.8.7),
-you'll need to install a fresh copy of Ruby.
+- **Don't Repeat Yourself:** DRY is a principle of software development which states that "Every piece of knowledge must have a single, unambiguous, authoritative representation within a system". By not writing the same information over and over again, our code is more maintainable, more extensible, and less buggy.
+- **Convention Over Configuration:** Rails has opinions about the best way to do many things in a web application, and defaults to this set of conventions, rather than require that you define them yourself through endless configuration files.
 
-To install Rails on Windows, you'll first need to install [Ruby Installer](https://rubyinstaller.org/).
+Creating a new Rails app
+------------------------
 
-For more installation methods for most Operating Systems take a look at
-[ruby-lang.org](https://www.ruby-lang.org/en/documentation/installation/).
+We're going to build a project called `store` that will be a simple e-commerce example app that showcases several powerful features that Rails includes as standard.
 
-#### Installing SQLite3
+### Prerequisites
 
-You will also need an installation of the SQLite3 database.
-Many popular UNIX-like OSes ship with an acceptable version of SQLite3.
-Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org).
+For this project, you will need:
 
-Verify that it is correctly installed and in your load `PATH`:
+* Ruby 3.2 or newer
+* Rails 8.0.0 or newer
 
-```bash
-$ sqlite3 --version
-```
-
-The program should report its version.
-
-#### Installing Rails
-
-To install Rails, use the `gem install` command provided by RubyGems:
-
-```bash
-$ gem install rails
-```
+Follow the [Install Ruby on Rails Guide](install_ruby_on_rails.html) if you need to install Ruby on Rails.
 
```

Comment:
We recommend they use WSL on Windows (not the old way of installing Ruby directly on Windows which would be C:\) so they will be in an Ubuntu terminal in Windows, so it will be the same for everyone.